iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -100,14 +100,12 @@ namespace popo
 using WaitSetIsConditionSatisfiedCallback = cxx::ConstMethodCallback<bool>;
 }
 constexpr uint32_t MAX_NUMBER_OF_CONDITION_VARIABLES = 1024U;
-constexpr uint32_t MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE = 128U;
-constexpr uint32_t MAX_NUMBER_OF_ATTACHMENTS_PER_WAITSET = 128U;
-static_assert(MAX_NUMBER_OF_ATTACHMENTS_PER_WAITSET <= MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE,
-              "The WaitSet capacity is restricted by the maximum amount of notifiers per condition variable.");
-// Listener
-constexpr uint8_t MAX_NUMBER_OF_EVENTS_PER_LISTENER = 128U;
-static_assert(MAX_NUMBER_OF_EVENTS_PER_LISTENER <= MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE,
-              "The Listener capacity is restricted by the maximum amount of notifiers per condition variable.");
+
+constexpr uint32_t MAX_NUMBER_OF_NOTIFIERS = build::IOX_MAX_NUMBER_OF_NOTIFIERS;
```

Comment:
Why was the change only done for `popo::Listener` and not for `popo::WaitSet`?

Reply:
I do not understand? This number is applied to waitset and listener attachments two lines below?

The goal is to make this configurable via cmake switch.